<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainWindow</class>
 <widget class="QMainWindow" name="AppMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="otbWrapperQtWidgetIcons.qrc">
    <normaloff>:/Utilities/Data/Icons/monteverdi-48x48.png</normaloff>:/Utilities/Data/Icons/monteverdi-48x48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>866</width>
         <height>454</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="message">
      <property name="text">
       <string>Status text</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="otb::Wrapper::QtWidgetSimpleProgressReport" name="progressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="executeButton">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionView_log"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionLoad_from_XML"/>
    <addaction name="actionSave_to_XML"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuParameters"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 87, 83);</string>
   </property>
  </widget>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAboutOTB">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionView_log">
   <property name="text">
    <string>View log</string>
   </property>
  </action>
  <action name="actionLoad_parameters">
   <property name="text">
    <string>Load parameters</string>
   </property>
  </action>
  <action name="actionSave_parameters">
   <property name="text">
    <string>Save parameters</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReset_parameters">
   <property name="text">
    <string>Reset parameters</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset All</string>
   </property>
  </action>
  <action name="actionLoad_from_XML">
   <property name="text">
    <string>Load from XML</string>
   </property>
  </action>
  <action name="actionSave_to_XML">
   <property name="text">
    <string>Save to XML</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>otb::Wrapper::QtWidgetSimpleProgressReport</class>
   <extends>QProgressBar</extends>
   <header>otbWrapperQtWidgetSimpleProgressReport.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="otbWrapperQtWidgetIcons.qrc"/>
 </resources>
 <connections/>
</ui>
